cypress/obj/variables.js

```diff
@@ -0,0 +1,59 @@
+class Variables {
```

Comment:
Why is this names Variables? This does not mean anything to anyone. It should be specific to your page that you are working on: LoginPage at the moment